Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/WebViewControlContentLoadingEventArgs.cs

```diff
@@ -10,48 +10,74 @@ namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT
     /// <summary>
     /// Provides data for the <see cref="IWebView.ContentLoading"/> and <see cref="IWebView.FrameContentLoading"/> events. This class cannot be inherited.
     /// </summary>
-    /// <remarks>Copy from <see cref="Windows.Web.UI.WebViewControlContentLoadingEventArgs"/> to avoid requirement to link Windows.winmd</remarks>
-    /// <seealso cref="Windows.Web.UI.WebViewControlContentLoadingEventArgs"/>
+    /// <remarks>Copy from <see cref="global::Windows.Web.UI.WebViewControlContentLoadingEventArgs"/> to avoid requirement to link Windows.winmd</remarks>
```

Comment:
Why make this change universally?

Reply:
Though, I should also say, it wasn't the only reason...  perhaps I have some misconfiguration on my dev box somewhere, but I couldn't compile without the global:: alias on all the WinMD refs, anyway.